cephalopod/factory.py

```diff
@@ -68,3 +62,17 @@ def __call__(self, *args, **kwargs):
 
     celery.Task = ContextTask
     return celery
+
+
+@multipass.identity_handler
+def identity_handler(identity_info):
+    """Implements the Flask-Multipass identity handler"""
+    cephalopod_id = identity_info.data['cephalopod_id']
```

Comment:
Just use `identity_info.identifier` and change the config accordingly (e.g. use the username on github instead of the id).

`cephalopod_id` isn't needed at all then..